src/Controls/src/Core/Platform/Android/DragAndDropGestureHandler.cs

```diff
@@ -272,10 +272,11 @@ public void OnLongPress(MotionEvent e)
 
 				// TODO MAUI
 				string clipDescription = String.Empty;//AutomationPropertiesProvider.ConcatenateNameAndHelpText(element) ?? String.Empty;
+				ClipData data = null;
 				ClipData.Item item = null;
```

Comment:
Item does not appear to be needed this high up, so maybe we can move it in the if block? Or further in if it can.

Reply:
Moved!